frontend/src/pages/AboutUs/AboutUsPage.module.scss

```diff
@@ -24,16 +24,17 @@
 
 // For primary-themed buttons
 .PrimaryButton {
-  background-color: #0b5240;
```

Comment:
Verifying jic so I know I'm understanding the changes -- these are already applied via Material Button styles right?

Reply:
Yes, the material components automatically apply certain styles based on the Theme. For instance background and font colors. You can reconfigure defaults in the Theme: https://github.com/SatcherInstitute/health-equity-tracker/blob/master/frontend/src/styles/MaterialTheme.tsx

If you're overriding Material component styles in the stylesheet in a class, sometimes you have to put !important at the end to make sure they're prioritized over the defaults.